### PR TITLE
build: enable python 3.6 in CI tests

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: tests/unit/_internal/test_store.py [hunk 1]
```diff
@@ -12,6 +13,11 @@
 from bentoml._internal.store import StoreItem
 from bentoml._internal.types import Tag
 
+if sys.version_info < (3, 7):
+    from backports.datetime_fromisoformat import MonkeyPatch
```
Comment (aarnphm) [inline tests/unit/_internal/test_store.py:17]: @sauyon

Comment (sauyon) [reply tests/unit/_internal/test_store.py:17]: Seems fine to me?

Comment (aarnphm) [reply tests/unit/_internal/test_store.py:17]: I will merge this in then

Comment (aarnphm) [reply tests/unit/_internal/test_store.py:17]: We will probably have to wait from users feedback if there are any performance issue

**Review by aarnphm — APPROVED** [review]

**Pull request merged**